TidepoolKit/Model/TInsulin.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  TInsulin.swift
+//  TidepoolKit
+//
+//  Created by Darin Krauss on 3/9/20.
+//  Copyright © 2020 Tidepool Project. All rights reserved.
+//
+
+public struct TInsulin {
+    public enum Units: String, Codable {
+        case units = "Units"
+    }
```

Comment:
hm... I would have expected `IU` (international units). Does Units = IU? Are these synonymous in this space?

Reply:
As I understand, the term "Unit" (with this capitalization) is the officially recognized unit of insulin measurement and what should be used for all scholarly papers, technical writing, etc. I can find the source if you'd like, but it's what we've been using for a while now.